Dockerfile

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
```

Comment:
Would you be willing to assign copyright to `Cisco Systems, Inc. and/or its affiliates. All rights reserved.`?

Reply:
You mean as a single line, or in addition to?

Reply:
I'm going to check in with our open source lawyer to find out what is required, and to confirm if we need you to sign a CLA. I also just asked about the CLA bot idea.  I'll let you know when I have an answer.

Reply:
thank you. Traditionally having multiple's is ok; but it indeed depends on what your lawyer says :)

Reply:
I spoke with the lawyer.  It sounds like having multiple is the way to go.  And there's no need for the CLA in this case, mostly because this isn't a new feature going into clamav itself, it's more like build system tooling.

Reply:
oh i totally missed this reply. Perfect, thanks for getting that sorted! And allowing for eternal fame j/k :p